### ISSUE TITLE
`interface { interface{} }` leads to erroneous warnings

### ISSUE THREAD

**Issue by d-enk** [tooltitude/support#41]
Tooltitude Version: `v0.74.0`

OS Version: `OS: Linux x64 6.5.0-25-generic`

VS Code Version: `1.89.1`

Go Version: `1.22.3`

```go
package main

func zero[T any]() (t T) { return }

func main() {
	zero1 := zero[int]() // Unused Write

	zero2 := zero[interface { // Unused Write
		interface{}
	}]()

	panic([]any{zero1, zero2})
}

```

```go.mod
//go.mod
module test

go 1.22.3
```

**Comment by tooltitude-support** [comment]
Thanks for the feedback. Will take a look at it a bit later today!

**Comment by tooltitude-support** [comment]
It's a parse error, i.e. interface { interface {}} doesn't seem to be parsed correctly.

**Comment by tooltitude-support** [comment]
Ok. Fixed it. The fix will be out in 0.75, currently planned to be out around 05/19.

**Comment by tooltitude-support** [comment]
0.75.0 is out. Please let us know if you have this problem again with it.  Closing issue assuming that it's fixed.
